### PR TITLE
chore: prepare release 0.12.2

### PR DESCRIPTION
**Pull request by jachro**
/deploy #persist

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,10 @@
 .. _changelog:
 
+0.12.2
+------
+
+Minor release to fix various bugs across the components.
```
Comment (ableuler) [inline CHANGELOG.rst:6]: Please add one more sentence about the bugs which were fixed and/or the affected components.

**Review by ableuler — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (jachro) [reply CHANGELOG.rst]: Done. Please take a look @ableuler.

Comment (rokroskar) [reply CHANGELOG.rst]: normally we'd link to a specific release or changelog from the individual components. I realize now this wasn't done for 0.12.1 either, so "minor bug fixes" is pretty ambiguous.

Comment (jachro) [reply CHANGELOG.rst]: Added links to releases.

Comment (rokroskar) [reply CHANGELOG.rst]: great, thanks!

Comment (rokroskar) [reply CHANGELOG.rst]: @olevski do we need to add a blurb about the problems with renku-core metrics?

**Review by ableuler — APPROVED** [review]
LGTM

File: CHANGELOG.rst (hunk 1 above)
Comment (jachro) [reply CHANGELOG.rst]: @rokroskar sorry, I've closed the PR already. I hope no more changes are needed...

**Comment by rokroskar** [comment]
@ableuler @olevski doesn't renku-core 1.1.0 break if the metrics are enabled?

**Comment by olevski** [comment]
> @ableuler @olevski doesn't renku-core 1.1.0 break if the metrics are enabled?

Yes it does. I will add a note. Sorry I just saw this.

**Pull request merged**